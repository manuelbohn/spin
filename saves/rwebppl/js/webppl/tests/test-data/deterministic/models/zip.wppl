[
  zip([], []),
  zip(['a', 'b', 'c'], [1, 2, 3]),
  sum(map(sum,
          zip(repeat(50000, function() { return 0 }),
              repeat(50000, function() { return 1 }))))
]
