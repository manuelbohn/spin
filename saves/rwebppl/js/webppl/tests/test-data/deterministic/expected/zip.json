{
  "result": [
    [],
    [["a",1],["b",2],["c",3]],
    50000
  ]

}
